demo_interface_select_folder.py

```
@@ -0,0 +1,232 @@
+import json
+from tkinter import *
```

Ruff (0.8.2)

`from tkinter import *` used; unable to detect undefined names

(F403)